apps/server/src/modules/group/controller/dto/response/group.response.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { ExternalSourceResponse } from './external-source.response';
+import { GroupTypeResponse } from './group-type.response';
+import { GroupUserResponse } from './group-user.response';
+
+export class GroupResponse {
+	@ApiProperty()
+	id: string;
+
+	@ApiProperty()
+	name: string;
+
+	@ApiProperty()
```

Comment:
I think you have to declare that this is an enum in the ApiProperty. Maybe check the types that are generated by the api generator

Reply:
Yes, you re right :)